src/content/docs/dropins/checkout/checkout-installation.mdx

```diff
@@ -1,9 +1,90 @@
 ---
-title: Checkout Installation
-description: Learn how to install the Checkout dropin into your site.
-sidebar:
-  # label: Checkout Installation
-  order: 2
+title: Checkout installation
+description: Learn how to install and initialize the Checkout dropin on your site.
 ---
 
-Learn how to install the Checkout dropin into your site.
+import OptionsTable from '@components/OptionsTable.astro';
+
+The Checkout dropin provides a customizable UI for the checkout process. The dropin is designed to be integrated into a your storefront and provides a seamless checkout experience for customers.
+
+## Installation
+
+Since the Checkout dropin relies on containers from several other dropins, you must install and configure these dropins before you can use the Checkout dropin.
+
+The [Commerce boilerplate template](https://github.com/hlxsites/aem-boilerplate-commerce) includes all of the necessary dropins and configurations to help you get started quickly, so Adobe recommends relying on the boilerplate instead of installing, configuring, and integrating the dropins individually.
+
+## Admin configuration
+
+Before you can use the Checkout dropin on your storefront, you must enable and configure [payment providers](https://experienceleague.adobe.com/en/docs/commerce-admin/stores-sales/payments/payments) and [checkout options](https://experienceleague.adobe.com/en/docs/commerce-admin/stores-sales/point-of-purchase/checkout/checkout-process) in the Adobe Commerce Admin.
+
+:::note
+The Checkout dropin [overview](/dropins/checkout/) provides a summary of supported Adobe Commerce features.
+:::
+
+## Initializer configuration
```

Comment:
Need to move this to the Checkout functions page.

Reply:
Actually, will either leave here or create a separate page to match what we did for PDP recently.